src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,4 +1,9 @@
-// Add the required types and props
-export const UserInfo = () => (
-  <>UserInfo markup</>
+import { Todo } from '../../types/Todo';
+
+type Props = Pick<Todo, 'user'>;
+
+export const UserInfo: React.FC<Props> = ({ user }) => (
+  <a className="UserInfo" href={'mailto:' + user?.email}>
+    {user?.name}
```

Comment:
what type User?
if ?name - don't change
But if the user always has name - change this plz
```suggestion
    {user.name}
```